tests_integration/test_end_to_end.py

```diff
@@ -171,6 +174,95 @@ def test_valid_miseq_upload(self):
+        we expect to see batch01-1111 and batch03-3333 uploaded
+        :return:
+        """
+        # Set our sample config file to use miseq parser and the correct irida credentials
+        self.write_to_config_file(
+            client_id=tests_integration.client_id,
+            client_secret=tests_integration.client_secret,
+            username=tests_integration.username,
+            password=tests_integration.password,
+            base_url=tests_integration.base_url,
+            parser="miseq"
+        )
+
+        # instance an api
+        test_api = api.ApiCalls(
+            client_id=tests_integration.client_id,
+            client_secret=tests_integration.client_secret,
+            base_url=tests_integration.base_url,
+            username=tests_integration.username,
+            password=tests_integration.password
+        )
+
+        # Create a test project, the uploader does not make new projects on its own
+        # so one must exist to upload samples into
+        # This may not be the project that the files get uploaded to,
+        # but one will be made in the case this is the only test being run
+        project_name = "test_batch_project"
+        project_description = "test_batch_project_description"
+        project = model.Project(name=project_name, description=project_description)
+        test_api.send_project(project)
+        # We always upload to project "1" so that tests will be consistent no matter how many / which tests are run
+        project_id = "1"
+
+        # Do the upload
+        upload_result = batch_upload_single_entry(path.join(path_to_module, "fake_batch_data"))
+
+        # Make sure the upload was a success
+        self.assertEqual(upload_result, 0)
+
+        # Verify the files were uploaded
+        sample_list = test_api.get_samples(project_id)
+
+        sample_1_found = False
+        sample_2_found = False
+        sample_3_found = False
+
+        for sample in sample_list:
+            if sample.sample_name in ["batch01-1111", "batch02-2222", "batch03-3333"]:
+                if sample.sample_name == "batch01-1111":
+                    sample_1_found = True
+                    sequence_files = test_api.get_sequence_files(project_id, sample.sample_name)
+                    self.assertEqual(len(sequence_files), 2)
+                    res_sequence_file_names = [
+                        sequence_files[0]['fileName'],
+                        sequence_files[1]['fileName']
+                    ]
+                    expected_sequence_file_names = [
+                        'batch01-1111_S1_L001_R1_001.fastq.gz',
+                        'batch01-1111_S1_L001_R2_001.fastq.gz'
+                    ]
+                    self.assertEqual(res_sequence_file_names.sort(), expected_sequence_file_names.sort())
+                elif sample.sample_name == "batch02-2222":
+                    # this one should not be found
+                    sample_2_found = True
```

Comment:
Change this to sample_2_not_found = False

Reply:
Fixed in effa23ba966ebf6801960a9a3075812ff0446a91